lms/static/js/verify_student/views/make_payment_step_view.js

```diff
@@ -105,10 +105,20 @@ var edx = edx || {};
                 self._getProductText( templateContext.courseModeSlug, templateContext.upgrade )
             );
 
-            // create a button for each payment processor
-            _.each(processors.reverse(), function(processorName) {
-                $( 'div.payment-buttons' ).append( self._getPaymentButtonHtml(processorName) );
-            });
+            if (processors.length === 0) {
+                // No payment processors are enabled at the moment, so show an error message
+                this.errorModel.set({
+                    errorTitle: gettext('No available payment processors'),
```

Comment:
@peter-fogg, @srpearce may have a better suggestion, but we might want to expose a phrase other than "payment processors" to the user. What about something like "We're having trouble accepting payments"?